wandb/sdk/internal/internal_api.py

```diff
@@ -1312,7 +1322,9 @@ def no_retry_4xx(e):
             body = json.loads(e.response.content)
             raise UsageError(body["errors"][0]["message"])
 
-        for mutation in mutation_new, mutation_old:
+        mutations = [mutation_3, mutation_2, mutation_1]
```

Comment:
This is ok, but let's add a comment TODO to get rid of this.  We can query the server for it's version and decide what functionality to use explicitly instead of retrying requests.

Reply:
done